pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -18,10 +18,10 @@ import (
 )
 
 var info = capabilities.MustNewCapabilityInfo(
-	"ocr3",
+	"offchain_reporting",
 	capabilities.CapabilityTypeConsensus,
 	"OCR3 consensus exposed as a capability.",
-	"v1.0.0",
+	"v3.0.0",
```

Comment:
This is exposed to end users, so the versioning is different from our internal versioning. I suggest we stick with v1

Reply:
OCR versioning is also somewhat public (we release papers or give public talks about OCR3). If we refer to OCR I'd prefer to use v3. An alternative is to call is generically "consensus" and then we can version independently.